src/css/components/grid.scss

```diff
@@ -24,3 +24,82 @@
     }
   }
 }
+
+/* stylelint-disable declaration-no-important  */
+%row {
```

Comment:
The amount of `!important` in this file sounds like a bad idea. By looking at it, I can't really see why there's a need for important.